packages/i18n/src/i18n.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * External dependencies
+ */
+import Tannin from 'tannin';
+
+/**
+ * @typedef {{[key: string]: any}} LocaleData
+ */
+
+/**
+ * Default locale data to use for Tannin domain when not otherwise provided.
+ * Assumes an English plural forms expression.
+ *
+ * @type {LocaleData}
+ */
+const DEFAULT_LOCALE_DATA = {
+	'': {
+		plural_forms: ( n ) => ( n === 1 ? 0 : 1 ),
+	},
+};
+
+/**
+ * Instantiable I18N methods
+ */
+export class I18N {
+	constructor() {
```

Comment:
It would be nice to instatiate the instance with locale data instead of instantiating then calling `setLocaleData`.